src/jsMain/kotlin/app/softwork/routingcompose/Router.kt

```diff
@@ -2,14 +2,35 @@ package app.softwork.routingcompose
 
 import androidx.compose.runtime.*
 
+/**
+ * Provide the router implementation through a CompositionLocal so deeper level
+ * Composables in the composition can have access to the current router.
+ *
+ * This is particularly useful for [NavLink], so we can have a single Composable
+ * agnostic of the top level router implementation.
+ */
+public val RouterCompositionLocal: ProvidableCompositionLocal<Router> =
+    staticCompositionLocalOf { error("Stupid error") }
```

Comment:
Please change the error description, although this error should never be seen by the user :)

Reply:
🤦  Whoops, was using this for local testing and completely forgot to clean this up!